backend/src/main/java/ba/unsa/etf/si/bbqms/admin_service/implementation/DefaultStationService.java

```diff
@@ -128,4 +128,14 @@ public Set<TellerStation> getAllOfferingService(final Service service) {
     public Optional<TellerStation> findById(final long serviceId) {
         return this.tellerStationRepository.findById(serviceId);
     }
+
+    @Override
+    public Set<Service> getServicesForTellerStation(long stationId) {
+        TellerStation tellerStation = tellerStationRepository.findById(stationId).orElse(null);
```

Comment:
If a function takes parameters, we should use `this` when accessing class attributes. 
So, `this.tellerStationRepository.findById(stationId).orElse(null);`